src/components/views/rooms/EventTile.js

```diff
@@ -366,10 +366,11 @@ module.exports = WithMatrixClient(React.createClass({
     },
 
     onCryptoClicked: function(e) {
-        var EncryptedEventDialog = sdk.getComponent("dialogs.EncryptedEventDialog");
         var event = this.props.mxEvent;
 
-        Modal.createDialog(EncryptedEventDialog, {
+        Modal.createDialogAsync((cb) => {
+            require(['../../../async-components/views/dialogs/EncryptedEventDialog'], cb)
```

Comment:
agreed, though where have we used commonjs elsewhere? (And it's the callback that makes the difference between the two types of `require`, rather than the arrayness, but yes)

will make it so